src/deepak/assignment23/PrimeNumberOperations.java

```diff
@@ -0,0 +1,45 @@
+/*Assignment_23 : 20th Nov'2020
+1) Find count of prime numbers between 2-100.
+2) find sum of all prime numbers between 2-100.
+3) find average of all prime numbers between 2-100 */
+
+
+package deepak.assignment23;
+
+import java.util.Scanner;
+
+public class PrimeNumberOperations {
+
+	void avgSumPrimeNumbers(int start, int end) {
+		int count=0,sum=0;
+		float avg=0f;
+		if(start > end || start == end)
+			System.out.println("Please enter correct range");
+		else {
+			for(int index=start;index<=end;index++) {
+				boolean flag = true;
+				for(int innerIndex=2;innerIndex<index;innerIndex++) {
```

Comment:
you should iterate till square root of index value, this will satisfies conditions.
Math.sqrt(index)